transmart-rest-api/grails-app/controllers/org/transmartproject/rest/RestExportController.groovy

```diff
@@ -0,0 +1,241 @@
+package org.transmartproject.rest
+
+import grails.converters.JSON
+import org.grails.web.converters.exceptions.ConverterException
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestParam
+import org.transmartproject.core.exceptions.AccessDeniedException
+import org.transmartproject.core.exceptions.InvalidArgumentsException
+import org.transmartproject.core.users.UsersResource
+import org.transmartproject.db.user.User
+import org.transmartproject.rest.dataExport.RestExportService
+import org.transmartproject.rest.misc.CurrentUser
+import com.recomdata.transmart.domain.i2b2.AsyncJob
+
+class RestExportController {
+    
+    @Autowired
+    RestExportService restExportService
+    @Autowired
+    CurrentUser currentUser
+    @Autowired
+    UsersResource usersResource
+
+    static def globalParams = [
+            "controller",
+            "action",
+            "format",
+            "apiVersion"
+    ]
+
+    /**
+     * Create a new asynchronous dataExport job:
+     * <code>/v2/export/job?jobName=${jobName}</code>
+     *
+     * If the ${jobName} is not specified, a default name will be created.
+     * ${jobName} = username-jobtype-ID
+     *
+     * @param jobName - optional
+     * @return {@link AsyncJob} instance
+     */
+    def createJob(@RequestParam('jobName') String jobName) {
+        checkParams(params, ['jobName'])
+
+        User user = (User) usersResource.getUserFromUsername(currentUser.username)
+        checkJobNameUnique(jobName)
+
+        def instance = restExportService.createExportJob(user, jobName)
+        respond instance
```

Comment:
Maybe it is better to add some wrapper here, but I am not sure what exactly need to be returned.